tools/design-tokens/shared/BUILD.bazel

```diff
@@ -0,0 +1,13 @@
+load("@npm//@bazel/typescript:index.bzl", "ts_library")
+
+package(default_visibility = ["//:__subpackages__"])
```

Comment:
This doesn't seem to work since we're using the build targets of both `generate-color-palette` and `design-tokens` in `libs/shared/design-tokens/BUILD.bazel`